.travis.yml

```diff
@@ -190,9 +190,10 @@ script:
     fi
   - |
     if [[ -v KUDU_TESTS ]]; then
-      presto-kudu/bin/run_kudu_tests.sh 3 null
-      presto-kudu/bin/run_kudu_tests.sh 1 ""
-      presto-kudu/bin/run_kudu_tests.sh 1 presto::
+      presto-kudu/bin/run_kudu_tests.sh 3 null &&
+      presto-kudu/bin/run_kudu_tests.sh 1 "" &&
+      presto-kudu/bin/run_kudu_tests.sh 1 presto:: &&
```

Comment:
[nit] the convention in file would suggest using separate `if` block per each command